stories/core/ErrorPage.stories.tsx

```diff
@@ -128,20 +128,20 @@ customHtmlMessage.args = {
   ),
 };
 
-export const withoutMessageOrButtons: Story<ErrorPageProps> = (props) => (
+export const WithoutMessageOrButtons: Story<ErrorPageProps> = (props) => (
   <ErrorPage {...props} />
 );
-withoutMessageOrButtons.args = {
+WithoutMessageOrButtons.args = {
   primaryButtonLabel: undefined,
   secondaryButtonLabel: undefined,
   primaryButtonHandle: undefined,
   secondaryButtonHandle: undefined,
 };
 
-export const translatedMessages: Story<ErrorPageProps> = (props) => (
+export const TranslatedMessages: Story<ErrorPageProps> = (props) => (
   <ErrorPage {...props} />
 );
-translatedMessages.args = {
+TranslatedMessages.args = {
   errorType: 'generic',
   errorMessage:
     "We can't find the iModel that you are looking for or it does not exist. Visit the iModel HUB or contact our support team.",
```

Comment:
Random: why are we using "fOrBiDdEn" as different language? Why not actual language like Spanish or French?

Reply:
Ask the person who wrote that code 😄  But I don't want to change that in this PR

Reply:
> > Expected version "^12.22.0 || ^14.17.0 || >=16.0.0". Got "14.16.1"
> 
> Well, it says right there. You need to update your node version 😄
> 
> I recommend using [`nvm`](https://github.com/nvm-sh/nvm) to get the latest 16.14.2

But it is weird, that linter package requires certain node version.. And then it also should appear in our documentation somewhere...
So not nice :/

Reply:
Oh, sorry.. I dont know how I commented in here versus reply in conversation :DD